src/Application/PlexAccounts/CheckUsername/CheckUsernameTaskHandler.cs

```diff
@@ -0,0 +1,67 @@
+using Application.Contracts;
+using Data.Contracts;
+using FluentValidation;
+using Logging.Interface;
+
+namespace PlexRipper.Application.PlexAccounts.CheckUsername;
+
+public class CheckUsernameTaskValidator : AbstractValidator<CheckUsernameTaskRequest>
+{
+    #region Constructors
+
+    public CheckUsernameTaskValidator()
+    {
+        RuleFor(x => x.Username).NotEmpty();
+        RuleFor(x => x.Username).MinimumLength(5);
+    }
+
+    #endregion
+}
+
+public class CheckUsernameTaskHandler : IRequestHandler<CheckUsernameTaskRequest, Result<bool>>
+{
+    #region Fields
+
+    private readonly ILog _log;
+    private readonly IMediator _mediator;
+
+    #endregion
```

Comment:
Maybe a personal preference but you have a lot of regions in the code. As classes start to be smaller, they suddenly clutter the code a lot. Consider removing them
(Usually, if you really need regions, you have bigger problems to care about)

Reply:
Probably a remnant from my junior days where I organized the file layout with regions to keep track of everything. But you're right, if it's needed then your classes are to big so I will keep removing the regions.

Reply:
Exactly, they're a code smell usually. The good news is it's just changing a rule in your linter (or Cleanup in Rider)